docs/dictionary/command/create-widget.lcdoc

```diff
@@ -0,0 +1,44 @@
+Name: create widget
+
+Type: command
+
+Associations: widget
+
+Syntax: create [invisible] widget [<controlName>] as <widgetKind> [in <group>]
+
+Summary: Create a <widget> control of the specified <kind>.
+
+Introduced: 8.0
+
+OS: mac,windows,linux,ios,android
+
+Platforms: desktop,server,web,mobile
+
+Example:
+create widget "My Navbar" as "com.livecode.widget.navbar"
+
+Example:
+create group "Swatches"
+repeat with x = 1 to 20
+	create invisible widget ("Color Swatch" && x) as "com.livecode.widget.colorswatch" in group "Swatches"
+end repeat
+
+Parameters:
+controlName: The name to call the newly created <widget>. If you don't specify a name, the <widget> is created with a default name.
+widgetKind: The <kind> of <widget> to create. 
```

Comment:
Maybe something like, "The name of the module that defines the widget to be created"?